NICE.Identity.Authorisation.WebAPI/Migrations/IdentityContextModelSnapshot.cs

```diff
@@ -434,8 +436,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .WithMany("ArchivedUserEmailHistory")
                         .HasForeignKey("ArchivedByUserId")
                         .HasConstraintName("FK_UserEmailHistory_ArchivedByUser_Users")
-                        .OnDelete(DeleteBehavior.Restrict)
-                        .IsRequired();

```

Comment:
I'm not sure on this one. I think this file is generated automatically by codefirst migrations. It might be that the data object model has changed and I'm the first one to run the migrations update since. Its either that, or there is a new version of Entity framework which has come along and the way it interprets the model into code has changed.

This will need a bit more investigation.

Reply:
Confirmed with BA that this field is now required, made the appropriate code changes

Reply:
Should the IsRequired() be added back, now the cascading delete has been done?